Sources/TensorFlow/Operators/Basic.swift

```diff
@@ -1325,76 +1325,62 @@ extension Tensor.IndexPath {
 //===------------------------------------------------------------------------------------------===//
 
 extension Tensor {
-  /// Returns `true` if `axis` is in `-rank..<rank`, indicating it refers to an axis of `self`.
+  /// Returns `true` iff `axis` can refer to an axis of `self`.
```

Comment:
Nit 1,

```suggestion
  /// Returns `true` iff `axis` denotes an axis of `self`.
```

Reply:
Works for me! Took both. Thanks for the suggestions.